hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/CreatePrecompileSuite.java

```diff
@@ -115,16 +144,570 @@ List<HapiSpec> positiveSpecs() {
+                        getContractInfo(TOKEN_CREATE_CONTRACT)
+                                .has(ContractInfoAsserts.contractWith().autoRenewAccountId(ACCOUNT))
+                                .logged())
+                .when(withOpContext((spec, opLog) -> allRunFor(
+                        spec,
+                        contractCall(
+                                        TOKEN_CREATE_CONTRACT,
+                                        "createTokenWithKeysAndExpiry",
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getAccountID(ACCOUNT))),
+                                        ed2551Key.get(),
+                                        spec.registry()
+                                                .getKey(ECDSA_KEY)
+                                                .getECDSASecp256K1()
+                                                .toByteArray(),
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getContractId(TOKEN_CREATE_CONTRACT))),
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getContractId(TOKEN_CREATE_CONTRACT))),
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getAccountID(ACCOUNT))),
+                                        AUTO_RENEW_PERIOD,
+                                        HapiParserUtil.asHeadlongAddress(
+                                                asAddress(spec.registry().getAccountID(ACCOUNT_TO_ASSOCIATE))))
+                                .via(FIRST_CREATE_TXN)
+                                .gas(GAS_TO_OFFER)
+                                .sending(DEFAULT_AMOUNT_TO_SEND)
+                                .payingWith(ACCOUNT)
+                                .alsoSigningWithFullPrefix(THRESHOLD_KEY)
+                                .refusingEthConversion()
+                                .exposingResultTo(result -> {
+                                    log.info(EXPLICIT_CREATE_RESULT, result[0]);
+                                    final var res = (Address) result[0];
+                                    createTokenNum.set(res.value().longValueExact());
+                                })
+                                .hasKnownStatus(SUCCESS))))
+                .then(sourcing(() -> getTokenInfo(asTokenString(TokenID.newBuilder()
+                                .setTokenNum(createTokenNum.get())
+                                .build()))
+                        .logged()
+                        .hasAutoRenewAccount(ACCOUNT)
+                        .hasPauseStatus(TokenPauseStatus.Unpaused)));
+    }
+
+    // TEST-003
```

Comment:
Should this be TEST-004?

Reply:
No, since the function we are calling `createNFTTokenWithKeysAndExpiry` is annotates with    TEST-003, I think that the test have to be also the same.